src/types/ChargingStation.ts

```diff
@@ -150,6 +151,7 @@ export interface RemoteAuthorization {
   connectorId: number;
   tagId: string;
   timestamp: Date;
+  oicpIdentification?: OICPIdentification;
```

Comment:
Why do we have oicpIdentification here and not OCPI?
Does the handling of the auth ID is different?

Reply:
The difference is that OICP does not provide the TagID in case of a remoteAuthorization.

I receive the EvcoID inside a OICPIdentification object.
I guess I can put the EvcoID as tagID inside the RemoteAuthorization, but think that is a workaround that can cause problems

Reply:
Later in the process it is harder to identify if it was a remote authorization or not.
I need the OICPIdentification object for the authorizeStop request. That's why I put I in there.

Reply:
Ok